vtr_flow/tech/PTM_22nm/readme.txt

```diff
@@ -1,2 +1,2 @@
 This technology file was obtained here:
-http://ptm.asu.edu/modelcard/LP/22nm_LP.pm
+http://ptm.asu.edu/modeclard/LP/22nm_LP.pm
```

Comment:
Looks like this was a stray search and replace, as the original is a valid URL, but the new one returns 404.